python/graphstorm/gsf.py

```diff
@@ -542,6 +542,9 @@ def set_encoder(model, g, config, train_task):
                                           use_node_embeddings=config.use_node_embeddings,
                                           force_no_embeddings=config.construct_feat_ntype,
                                           num_ffn_layers_in_input=config.num_ffn_layers_in_input)
+    # The number of feature dimensions can change. For example, the feature dimensions
+    # of BERT embeddings are determined when the input encoder is created.
+    feat_size = encoder.in_dims
```

Comment:
It seems feat_size is not used in the following code.

Reply:
you can search for `feat_size` in the hidden code.